packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts

```diff
@@ -184,7 +184,7 @@ describeNoCompat("Batching failures", (getTestObjectProvider) => {
     });
 
     describe("client sends invalid batches ", () => {
-        itExpects("Batch end without start",
+        itExpects.skip("Batch end without start",
```

Comment:
Can you revert the other files, please?

Reply:
😲  how was this merged without a review?

Reply:
Why do the other files get reverted?

Reply:
> 😲 how was this merged without a review?

This isn't a merge into `next`, it's just a merge into [the branch that is going to be merged](https://github.com/microsoft/FluidFramework/pull/13567/files) into `next`.